modules/lvms-deleting-volume-snapshots.adoc

```diff
@@ -0,0 +1,40 @@
+// Module included in the following assemblies:
+//
+// storage/persistent_storage/persistent_storage_local/persistent-storage-using-lvms.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="lvms-deleting-volume-snapshots_{context}"]
+= Deleting volume snapshots
+
+You can delete the volume snapshots of the Persistent Volume Claims (PVCs).
+[IMPORTANT]
+====
+When you delete a persistent volume claim (PVC), {lvms} deletes only the PVC but not the snapshots of the PVC.
+====
+
+.Prerequisites
+
+. You have ensured that the volume snpashot that you want to delete is not in use.
+
+.Procedure
+
+. Log in to the OpenShift CLI (`oc`).
++
+[IMPORTANT]
+====
+If you are using {lvms} in {rh-rhacm}, log in to the {rh-rhacm} CLI using your {product-title} credentials.
+====
+
+. Delete the volume snapshot by running the following command:
++
+[source,terminal]
+----
+# oc delete volumesnapshot <volume_snapshot_name> -n <namespace>
+----
+
+. To delete the restored volume snapshot, delete the PVC that you created to restore the volume snapshot by running the following command:
++
+[source,terminal]
+----
+# oc delete pvc <pvc_name> -n <namespace>
```

Comment:
This would only delete the PV provisioned by that PVC, volumeSnapshot resource would still be available in the cluster.